component.json

```diff
@@ -253,6 +253,52 @@
           "prompt": "Please select a Event object"
         }
       }
+    },
+    "streamPlatformEventsPubSub": {
+      "title": "Subscribe to PubSub",
+      "order": 97,
+      "help": {
+        "description": "Can be used for subscription to the specified in the configuration Platform Event object. Can be used only for Realtime flows",
```

Comment:
```suggestion
        "description": "Can be used to subscribe to an object specified in the configuration Platform Event. **Can be used in Realtime flows only**",
```

Reply:
`Can be used to subscribe to an object specified in the configuration Platform Event.` - changed
`Can be used in Realtime flows only!` - removed (this one can work as ordinary)